tests/unit/test_check_operators/test_sql_containment_checks.py

```diff
@@ -0,0 +1,161 @@
+
+
+@pytest.mark.parametrize(
+    "data,comparator,value_is_literal,expected_result",
+    [
+        (
+            {"target": ["Ctt", "Btt", "A"]},
+            ["Ctt", "B", "A"],
+            True,
+            [False, True, False],
+        ),
+        (
+            {"target": ["A", "B", "C"]},
+            ["C", "Z", "A"],
+            True,
+            [False, True, False],
+        ),
+        (
+            {"target": ["A", "B", "C"], "VAR2": ["A", "B", "D"]},
+            "VAR2",
+            False,
+            [False, False, True],
+        ),
+        (
+            {"target": ["A", "B", "C"]},
+            "B",
+            True,
+            [True, False, True],
+        ),
+    ],
+)
+def test_is_not_contained_by(data, comparator, value_is_literal, expected_result):
+    table_name = "test_table"
+    tds = PostgresQLDataService.from_column_data(table_name=table_name, column_data=data)
+    sql_ops = PostgresQLOperators({"validation_dataset_id": table_name, "sql_data_service": tds})
+
+    result = sql_ops.is_not_contained_by(
+        {"target": "target", "comparator": comparator, "value_is_literal": value_is_literal}
+    )
+    assert result.equals(pd.Series(expected_result))
+
+
+@pytest.mark.parametrize(
+    "data,comparator,value_is_literal,expected_result",
+    [
+        (
+            {"target": ["Ctt", "Btt", "A"]},
+            ["ctt", "b", "a"],
+            True,
+            [True, False, True],
+        ),
+        (
+            {"target": ["A", "B", "C"]},
+            ["c", "z", "a"],
+            True,
+            [True, False, True],
+        ),
+        (
+            {"target": ["A", "B", "C"]},
+            "b",
+            True,
+            [False, True, False],
+        ),
+    ],
+)
+def test_is_contained_by_case_insensitive(data, comparator, value_is_literal, expected_result):
+    table_name = "test_table"
+    tds = PostgresQLDataService.from_column_data(table_name=table_name, column_data=data)
+    sql_ops = PostgresQLOperators({"validation_dataset_id": table_name, "sql_data_service": tds})
+
+    result = sql_ops.is_contained_by_case_insensitive(
+        {"target": "target", "comparator": comparator, "value_is_literal": value_is_literal}
+    )
+    assert result.equals(pd.Series(expected_result))
+
+
+@pytest.mark.parametrize(
+    "data,comparator,value_is_literal,expected_result",
+    [
+        (
+            {"target": ["Ctt", "Btt", "A"]},
+            ["ctt", "b", "a"],
+            True,
+            [False, True, False],
+        ),
+        (
+            {"target": ["A", "B", "C"]},
+            ["c", "z", "a"],
+            True,
+            [False, True, False],
+        ),
+        (
+            {"target": ["A", "B", "C"]},
+            "b",
+            True,
+            [True, False, True],
+        ),
+    ],
+)
+def test_is_not_contained_by_case_insensitive(data, comparator, value_is_literal, expected_result):
+    table_name = "test_table"
```

Comment:
Thought: Maybe we should be defining the inputs then just testing their NOT when doing the negative tests?